jsconf.json

```diff
@@ -0,0 +1,71 @@
+{
+    "tags": {
+        "allowUnknownTags| ": true,
+        "dictionaries": ["jsdoc","closure"]
+    },
+    "source": {
+        "include": [
+           "src/firefly/js/api/",
+           "src/firefly/js/tables/TableUtil.js",
+           "src/firefly/js/charts/XYPlotCntlr.js",
+           "src/firefly/js//visualize/CsysConverter.js",
+           "src/firefly/js/visualize/DrawLayerCntlr.js"
+          // ,"src/firefly/js/charts/HistogramCntlr.js"
```

Comment:
true, a way to do it is to put the '//' inside the double quote, that looks more as a comment.